stores/local_storage_store.test.jsx

```diff
@@ -32,4 +32,31 @@ describe('stores/LocalStorageStore', () => {
         assert.equal(LocalStorageStore.getPreviousChannelName(userId2, teamId1), channel2);
         assert.equal(LocalStorageStore.getPreviousChannelName(userId2, teamId2), channel3);
     });
+
+    test('should persist recent emojis per user', () => {
+        const userId1 = 'userId1';
+        const userId2 = 'userId2';
+        const recentEmojis1 = ['smile', 'joy', 'grin'].join(',');
```

Comment:
To properly testing as expected, this should be a list, instead of a string.

Reply:
Nice catch, didn't notice after recent change.  Now updated.  Thanks